src/java.management/share/classes/javax/management/modelmbean/DescriptorSupport.java

```diff
@@ -1120,6 +1128,7 @@ private static String makeFieldValue(Object value) {
      * - some other string, in which case the result is that string,
      * without the parentheses.
      */
+    @SuppressWarnings("removal")
```

Comment:
Can we add a comment to indicate that XMLParseException is deprecated for removal?